src/Core/Services/MetadataProviders/MsSqlMetadataProvider.cs

```diff
@@ -34,57 +32,12 @@ public override string GetDefaultSchemaName()
         }
 
         /// <summary>
-        /// Takes a string version of an MS SQL data type and returns its .NET common language runtime (CLR) counterpart
-        /// As per https://docs.microsoft.com/en-us/dotnet/framework/data/adonet/sql-server-data-type-mappings
+        /// Takes a string version of an SQL Server data type and returns its .NET common language runtime (CLR) counterpart
```

Comment:
Could be Azure SQL DB too, so SQL Server might be too specific.

Reply:
https://docs.microsoft.com/dotnet/framework/data/adonet/sql-server-data-type-mappings

Since there doesn't seem to be Azure SQL data types explicitly that differentiate from the defined Sql Server data types , I'm updating the comment to reflect that Sql Server data types _also apply to Azure SQL DB_